regulations/static/regulations/js/source/views/main/reg-view.js

```diff
@@ -179,6 +178,33 @@ var RegView = ChildView.extend({
         this.loadImages();
     },
 
+    openDef: function(defId, term, $link) {
+        // if its the same definition, diff term link
+        if ($('.open-definition').attr('id') === defId) {
+            this.toggleDefinition($link);
```

Comment:
General code concern: is this a function that is can be re-used elsewhere? As written, it is tightly coupled to $('.open-definition') element. Consider using DI here, Pros? Cons? Thoughts?